app/src/main/res/layout/fragment_news_detail.xml

```diff
@@ -0,0 +1,134 @@
+                    tools:src="@drawable/ic_back" />
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/static_spacing_medium_extra"
+                    android:text="@string/news_detail_post"
+                    android:textColor="@android:color/white"
+                    android:textSize="@dimen/font_large"
+                    android:textStyle="bold"
+                    tools:textColor="@android:color/black" />
+
+            </LinearLayout>
+
+            <android.support.constraint.Guideline
+                android:id="@+id/guideline_start"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:orientation="vertical"
+                fresco:layout_constraintGuide_begin="@dimen/static_spacing_medium_extra" />
+
+            <android.support.constraint.Guideline
+                android:id="@+id/guideline_end"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:orientation="vertical"
+                app:layout_constraintGuide_end="@dimen/static_spacing_medium_extra" />
+
+            <TextView
+                android:id="@+id/mTimeAgoText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentEnd="true"
+                android:layout_marginTop="@dimen/static_spacing_large"
+                android:gravity="center"
+                android:minWidth="@dimen/img_small"
+                android:textSize="@dimen/font_medium_more"
+                app:layout_constraintEnd_toEndOf="@id/guideline_end"
+                app:layout_constraintTop_toBottomOf="@id/mNewImage"
+                tools:text="15m" />
+
+            <TextView
+                android:id="@+id/mNewTitleText"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/static_spacing_large"
+                android:layout_marginEnd="@dimen/static_spacing_medium"
+                android:textColor="@color/colorPrimary"
+                android:textSize="@dimen/font_large_extra"
+                android:textStyle="bold"
+                app:layout_constraintEnd_toStartOf="@id/mTimeAgoText"
+                app:layout_constraintStart_toStartOf="@id/guideline_start"
+                app:layout_constraintTop_toBottomOf="@id/mNewImage"
+                tools:text="Ali Connors" />
+
+            <TextView
+                android:id="@+id/mNewText"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/static_spacing_medium_extra"
+                android:text="@string/design_placeholder_medium"
```

Comment:
Are you using this text to have a preview of the design?

Reply:
Yes, I forget to use `tools:text` that is only for preview purposes. I'll fix it.